src/Mod/TechDraw/Gui/DrawGuiUtil.cpp

```diff
@@ -127,17 +130,37 @@ TechDraw::DrawPage* DrawGuiUtil::findPage(Gui::Command* cmd)
                 page = qp->getDrawPage();
             } else {
                 // no active page
-                QMessageBox::warning(Gui::getMainWindow(), QObject::tr("Which page?"),
-                                     QObject::tr("Can not determine correct page."));
+                for(auto obj: selPages) {
```

Comment:
Just a trifle: we have no official style rules but try to use uniformly spaces behind for, if, else etc. In the file this style is already used.